expo-zustand-styled-components/src/components/FileViewer/FileViewer.tsx

```diff
@@ -12,8 +12,8 @@ import FileText from './FileText';
 
 import { ByteSize, Containter, Header, LineCount } from './FileViewer.styles';
 
-const FileViewer = () => {
-  const { path, owner, name, file, branch } = useRepoInfoStore();
+const FileViewer = ({ path, branch }: { path: string; branch: string }) => {
+  const { file, name, owner } = useRepoInfoStore();
```

Comment:
i belive the owner and name should be gotten from the URL params if we are really moving the stored values to the url

Reply:
![image](https://user-images.githubusercontent.com/36667083/227953513-2d0d8a12-c2cb-48a0-97d6-c2d5ec30cf0c.png)

From the image above the RepoNavigator has a path: ':owner/:name' and it has access to the name and owner path params, and we need to call getRepoInfo once. I noticed the subsequent screens don't have access to the owner and name path. not like the URL on the normal web, it is stored as JSON.